pkg/smbios/info_linux_amd64_test.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2016-2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build amd64 && linux
```

Comment:
note with this name, you don't need bulid tags, and I think it's good to use one or the other. When possible, I prefer the name, as an ls shows me the constraints.